angular-ui/src/app/app-routing.module.ts

```diff
@@ -7,7 +7,11 @@ import { ProfileComponent } from './profile/profile.component';
 import { UserLoginComponent } from './user-login/user-login.component';
 import { UserRegisterComponent } from './user-register/user-register.component';
 import { MyListComponent } from './my-list/my-list.component';
+<<<<<<< HEAD
```

Comment:
not sure why the merge conflict syntax stayed in here after you resolved conflicts. but you'll want to keep line 13 here and can delete the >>>>>> and ===== parts